cdk/src/open-data-platform/lambda/write-demographic-data-handler.ts

```diff
@@ -0,0 +1,233 @@
+// asset-input/src/open-data-platform/lambda/write-demographic-data-handler.js
+import {APIGatewayEvent, APIGatewayProxyCallback, Context} from 'aws-lambda';
+
+const AWS = require('aws-sdk');
+const parse = require('csv-parser');
+const S3 = new AWS.S3();
+const rdsDataService = new AWS.RDSDataService();
+
+const GEO_ID = 'GEOID';
+const RACE_TOTAL = 'RaceTotal';
+const WHITE_POPULATION = 'Estimate!!Total:!!White alone';
+const BLACK_POPULATION = 'Estimate!!Total:!!Black or African American alone';
+
+/// Prepared SQL statements.
+const CREATE_DEMOGRAPHICS_TABLE =
```

Comment:
Can you move this schema definition to [`schema.sql`](https://github.com/BlueConduit/open-data-platform/blob/main/cdk/src/open-data-platform/data-plane/schema.sql)? That should be the one spot where we define the schema for the DB.

Reply:
Done